test/iron-ajax.html

```diff
@@ -818,6 +824,39 @@
         });
       });
 
+      suite('when using the bubbles attribute', function () {
+        setup(function() {
+          server.restore();
+        });
+        test('the request and response events should bubble to window', function (done) {
+          server.restore();
+          var total = 0;
+          function incrementTotal(){
+            total++;
+            if(total === 1) done();
```

Comment:
nits: 
* project style is to always use curly braces with if statements, even if they're just one line
* prefer a space between `if` and the opening paren